minecraft_servers/wasd/manifest.json

```diff
@@ -0,0 +1,57 @@
+{
+  "server_name": "wasd",
+  "nice_name": "Wasd",
+  "direct_ip": "wasdmc.com",
+  "server_wildcards": [],
+  "supported_languages": [
+    "pt-BR",
+    "pt-PT"
+  ],
+  "social": {
+    "web": "https://loja.wasdmc.com",
+    "web_shop": "https://loja.wasdmc.com"
```

Comment:
use a specific link to your shop here

Reply:
This is still the case
Change to https://loja.wasdmc.com/itens